src/index.html

```diff
@@ -97,6 +103,9 @@
 
   <body>
     <div id="map"></div>
+    <p style="font-family: Overpass; font-weight: 600; visibility: hidden">
+      <b style="font-weight: 700">Invisible</b> text so the font will load early
+    </p>
```

Comment:
Do we need to set the ARIA role to [`presentation`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/presentation_role) or apply a `speak: none` property to force screen readers to skip this text?